test/presubmit-tests.sh

```diff
@@ -43,7 +46,21 @@ function unit_tests() {
 
 function integration_tests() {
   echo "Running integration tests"
-  echo "TODO(#16): add integration tests"
+
+  # Use knative test-infra scripts to make `fail-test` and `success` available.
+  # which will output helpful information for debugging failures, and ensure test
+  # results are made available.
+  source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.sh
+  # Setup the cluster (if not using an existing cluster) and deploy the Pipeline CRD
+  source $(dirname $0)/cluster-setup.sh
+
+  options=""
```

Comment:
If you update knative/test-infra to HEAD, you replace lines 57-61 with simply

`go_test_e2e ./test || fail_test`

Reply:
kk will do, thanks

Reply:
er wait, but then I'd have to source `e2e-tests.sh` in this file? I think I'm going to leave this for now, I've gotten nervous about what will happen when I source a script :sweat_smile: